static/app/components/replays/contextIcon.tsx

```diff
@@ -12,13 +12,14 @@ type Props = {
   name: string;
   version: undefined | string;
   className?: string;
+  hideVersion?: boolean;
 };
 
 const LazyContextIcon = lazy(
   () => import('sentry/components/events/contextSummary/contextIcon')
 );
 
-const ContextIcon = styled(({className, name, version}: Props) => {
```

Comment:
This might seem silly, but can we convert this to be in the affirmative: `showVersion`. 

We're generally inconsistent with this across all our components, but I find that the logic for negative boolean values is harder to reason about when things get complex.